akka-projection-mocks/src/main/scala/akka/projection/mocks/DBIO.scala

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.mocks
```

Comment:
looks more like stubs (or fakes) than mocks

https://www.martinfowler.com/articles/mocksArentStubs.html

Reply:
Yeah, it started as a `FakeDb` thing on #7. Then I want to reuse it for writing tests. Not sure yet if they will be real mocks or stubs. I guess it will stay as FakeDb that we can introspect to check if the events have been processed.

I will keep this in mind.